launcher/src/code-workspace.ts

```diff
@@ -30,54 +44,106 @@ export class CodeWorkspace {
       return;
     }
 
-    // skip if env.VSCODE_DEFAULT_WORKSPACE is defined and points to a real file
-    if (env.VSCODE_DEFAULT_WORKSPACE) {
-      if (
-        (await fs.pathExists(env.VSCODE_DEFAULT_WORKSPACE)) &&
-        (await fs.isFile(env.VSCODE_DEFAULT_WORKSPACE))
-      ) {
-        console.log(
-          "  > env.VSCODE_DEFAULT_WORKSPACE is defined, skip this step"
-        );
-        return;
-      } else {
-        console.log(
-          "  > env.VSCODE_DEFAULT_WORKSPACE must point on a workspace file"
-        );
-      }
-    }
+    // Folowing flow is common for default workspce location and
```

Comment:
Small typos: `Folowing` -> `Following`, `workspce` -> `workspace`

Reply:
Fast typing leads to many typos.
Thanks!